src/libraries/System.Private.CoreLib/src/System/Char.cs

```diff
@@ -1671,6 +1675,27 @@ static bool INumber<char>.TryParse(ReadOnlySpan<char> s, NumberStyles style, IFo
             return true;
         }
 
+        /// <inheritdoc cref="INumber{TSelf}.GetByteCount()" />
+        int INumber<char>.GetByteCount() => sizeof(char);
+
+        /// <inheritdoc cref="INumber{TSelf}.TryWriteLittleEndian(Span{byte}, out int)" />
+        bool INumber<char>.TryWriteLittleEndian(Span<byte> destination, out int bytesWritten)
+        {
+            if (destination.Length >= sizeof(char))
+            {
+                ushort value = BitConverter.IsLittleEndian ? m_value : BinaryPrimitives.ReverseEndianness(m_value);
+                Unsafe.WriteUnaligned(ref MemoryMarshal.GetReference(destination), value);
+
+                bytesWritten = sizeof(char);
+                return true;
+            }
+            else
+            {
+                bytesWritten = 0;
+                return false;
+            }
```

Comment:
Did you consider using BinaryPrimitives for these implementations?  I'm assuming you chose not to because those TryWriteXxLittleEndian methods don't take an out bytesWritten and thus you'd need to implement that manually anyway?

Reply:
Right, it would've done duplicate work/checks + not cover everything I need to do here. So it would ultimately not been any smaller or faster.